src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -363,30 +363,11 @@ Thumbnails =
 # Default: false
 ThumbnailPadding = 
 
-# Some renderers (e.g. Sony Bravia TVs and Blu-ray Disc players) require
-# thumbnails to be represented as JPEGs in the DLNA directory XML. Set this to
-# true to force UMS to fake this representation.
-# Note: this has no effect on real thumbnail file formats or content-type
-# headers, and will only return true if MediaInfo = true
-# Default: false
-ForceJPGThumbnails = 
```

Comment:
@SubJunk This works in a completely different way. UMS will offer any image in multiple formats and sizes, and the renderer will pick a suitable format. This is the way DLNA is designed to work, so chances are good that it will work much better than us having to configure it. 

I do expect that many renderers are "lazy" in that they will pick the first format they are offered that are compatible, so there's some logic to try to offer the most relevant thumbnail first. JPEG is the basic, mandatory image and thumbnail format in DLNA so it will be offered first unless the image has transparancy, in which case PNG will be offered first. Both formats will be offered regardless.